doc/radosgw/multisite.rst

```diff
@@ -0,0 +1,487 @@
+
+Deleting the default zonegroup
+------------------------------
+
+A simple installation of radosgw would assume the default zonegroup called
+"default", since we no longer need this we begin by removing the default
+zonegroup::
+
+  # radosgw-admin zonegroup delete --rgw-zonegroup=default
+
+
+Creating a master zonegroup
+---------------------------
+
+We'll be creating a zonegroup called ``us`` as a master zonegroup. A master
+zonegroup will be in control of the zonegroup map and propagate changes to the
+rest of the system. We will also set this zonegroup as the default, which allows
+explicitly mentioning the ``rgw-zonegroup`` switch for later commands.
+
+::
+
+   # radosgw-admin zonegroup create --rgw-zonegroup=us --endpoints=http://rgw1:80 --master
+   {
+    "id": "d4018b8d-8c0d-4072-8919-608726fa369e",
+    "name": "us",
+    "api_name": "us",
+    "is_master": "true",
+    "endpoints": [
+        "http:\/\/rgw1:80"
+    ],
+    "hostnames": [],
+    "hostnames_s3website": [],
+    "master_zone": "",
+    "zones": [],
+    "placement_targets": [],
+    "default_placement": "",
+    "realm_id": "4a367026-bd8f-40ee-b486-8212482ddcd7"
+    }
+
+Now we make this as the default zonegroup via the following command ::
+
+  # radosgw-admin zonegroup default --rgw-zonegroup=us
+
+
+Creating a master zone
+----------------------
+Next we create a zone, and make it as the default zone. Note that for metadata
+operations like user creation you would want to use this zone.
+
+::
+
+   #radosgw-admin zone create --rgw-zonegroup=us --rgw-zone=us-east --endpoints=http://rgw1:80 --access-key=$ZONE_ACCESS_KEY --secret=$ZONE_SECRET_KEY
+   {
+    "id": "83859a9a-9901-4f00-aa6d-285c777e10f0",
+    "name": "us-east",
+    "domain_root": "us-east.rgw.data.root",
+    "control_pool": "us-east.rgw.control",
+    "gc_pool": "us-east.rgw.gc",
+    "log_pool": "us-east.rgw.log",
+    "intent_log_pool": "us-east.rgw.intent-log",
+    "usage_log_pool": "us-east.rgw.usage",
+    "user_keys_pool": "us-east.rgw.users.keys",
+    "user_email_pool": "us-east.rgw.users.email",
+    "user_swift_pool": "us-east.rgw.users.swift",
+    "user_uid_pool": "us-east.rgw.users.uid",
+    "system_key": {
+        "access_key": "1555b35654ad1656d804",
+        "secret_key": "h7GhxuBLTrlhVUyxSPUKUV8r\/2EI4ngqJxD7iBdBYLhwluN30JaT3Q=="
+    },
+    "placement_pools": [
+        {
+            "key": "default-placement",
+            "val": {
+                "index_pool": "us-east.rgw.buckets.index",
+                "data_pool": "us-east.rgw.buckets.data",
+                "data_extra_pool": "us-east.rgw.buckets.non-ec",
+                "index_type": 0
+            }
+        }
+    ],
+    "metadata_heap": "us-east.rgw.meta",
+    "realm_id": "4a367026-bd8f-40ee-b486-8212482ddcd7"
+    }
+
+   # radosgw-admin zone default --rgw-zone=us-east
+
+Next we add the zone to the zonegroup::
+
+  # radosgw-admin zonegroup add --rgw-zonegroup=us --rgw-zone=us-east
+
+
```

Comment:
We allow adding the zone to the zonegroup in the create command by adding --rgw-zonegroup=<zg> and setting it as default by adding --default.